README.md

```diff
@@ -141,6 +141,20 @@ If you prefer an integrated IDE-like experience with autocompletion and instant
 preview, you can also check out the [Typst web app][app], which is currently in
 public beta.
 
+## Use docker
+To run prebuilt image inside docker run:
+
+```sh
+docker run -it --rm ghcr.io/typst/typst:main
```

Comment:
I'd like to prevent an overlage Readme. How about we merge this into the "Installing" section and keep only the first docker example?

Reply:
Ok